lib/utils.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
       throw new errors.FieldNotFoundError(parsedRecord, path);
     }
 
-    return value;
+    return value.toLowerCase();
```

Comment:
Not all fields need to be lowercased. I think, `indexField` needs a separate function that adds logic on top of the `getField`.

Reply:
Will take a look.